spec/services/notifications_list_spec.rb

```diff
@@ -61,6 +81,26 @@
 
         expect(NotificationsList.for(application_choice, include_ratifying_provider: true).to_a).to match_array([training_provider_user, ratifying_provider_user])
       end
+
+      it 'returns a provider user who is a member of the training and ratifying providers without duplicates' do
+        ratifying_provider = create(:provider)
+        training_provider = create(:provider)
+        provider_user = create(:provider_user, send_notifications: true, providers: [training_provider, ratifying_provider])
+        application_choice = create(:application_choice, course_option: create(:course_option, course: create(:course, provider: training_provider, accredited_provider: ratifying_provider)))
+
+        create(
+          :provider_user_notification_preferences,
+          :all_off,
+          provider_user: create(:provider_user, send_notifications: false, providers: [training_provider, ratifying_provider]),
+        )
+        create(:provider_user_notification_preferences)
+
```

Comment:
I think we can lose this whole block if the flag is off

Reply:
We added tickets to remove the feature flag next sprint, Im hoping we do the work then to avoid any confusion otherwise

Reply:
Sure. I'm not sure what the value is of having a symmetrical spec in the meantime?

Reply:
I'll leave the spec there for now without these two redundant lines, it's inside a _when the configurable provider notifications feature flag is off_ context  so it'll go away when the other flag off specs get dropped.